tests/e2e_decorrelation/test_null_semantics.py

```
@@ -0,0 +1,718 @@
+            SELECT * FROM pg.users
+            WHERE country IN ('US', 'UK')
+            ORDER BY id
+        """
+
+        parser = Parser()
+        binder = Binder(catalog)
+        decorrelator = Decorrelator()
+
+        logical_plan = parser.parse(sql)
+        bound_plan = binder.bind(logical_plan)
+        decorrelated_plan = decorrelator.decorrelate(bound_plan)
```

Ruff (0.14.5)

Local variable `decorrelated_plan` is assigned to but never used

Remove assignment to unused variable `decorrelated_plan`

(F841)